docs/src/modules/components/overview/Keyboard.tsx

```diff
@@ -0,0 +1,510 @@
+import * as React from 'react';
```

Comment:
"I noticed that when focusing out and re-entering an input through the keyboard, the focus on the field changes to previous sections. I think it would be expected for the focus to remain on the last focused section or to require the user to manually focus on a section before being able to edit it. WDYT?

https://github.com/user-attachments/assets/a449853c-a58a-4d65-a61b-afd14cc61e61

Reply:
I think this focus behavior is the default when you focus ->blur -> focus a field again 🤔 At least with keyboard navigation like in the video  

[Demo ](http://localhost:3001/x/react-date-pickers/custom-field/#using-material-pickerstextfield)where it's easy to test this 

https://github.com/user-attachments/assets/a3c718e0-d73e-4398-8721-4e2a1dec75d3